plugins/hls-retrie-plugin/src/Ide/Plugin/Retrie.hs

```diff
@@ -762,7 +761,7 @@ reuseParsedModule state f = do
 
 getCPPmodule :: Recorder (WithPriority Log) -> IdeState -> HscEnv -> FilePath -> IO (FixityEnv, CPP AnnotatedModule)
 getCPPmodule recorder state session t = do
-    nt <- toNormalizedFilePath' <$> makeAbsolute t
+    let nt = toNormalizedFilePath' $ (toAbsolute $ rootDir state) t
```

Comment:
We have rare usage of rootDir, this is one of it.

Reply:
This one is comming GHC, it should depend on how we load the file into ghc, I think we can drop this if we already loading the file into ghc with absolute path.